app/views/games/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <h1><%= @game.name %></h1>
 <p>Players: <%= @game.min_players %> to <%= @game.max_players %></p>
 <p>Playing time: <%= @game.playing_time %></p>
-<p>Complexity: <%= @game.complexity %></p>
+<p>Complexity: <%= Game::COMPLEXITIES[@game.complexity] %></p>
```

Comment:
This might be a good place for a method on the `Game` model:

```ruby
class Game < ActiveRecord::Base
  def complexity_name
    COMPLEXITIES[complexity]
  end
end
```